GREEDY.UnitTests/CreatePathForDataControllerUnitTest.cs

```diff
@@ -0,0 +1,80 @@
+﻿using GREEDY.Controllers;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+using System.Configuration;
+
+namespace GREEDY.UnitTests
+{
+    [TestClass]
+    public class CreatePathForDataControllerUnitTest
+    {
+
+        private const string FilePath = "../../../Data/receipts/";
+
+        [TestMethod]
+        [Description("Check to see if a folder is created when we create it first time")]
+
+        [DataRow(FilePath)]
+
+        public void CreateFolder_FirstTimeCreating_FolderCreated(string data)
+        {
+            //arrange
+            CreatePathForDataController createPath = new CreatePathForDataController();
+            DirectoryInfo folderDirectory = new DirectoryInfo(data);
+
+            //act
+            if (Directory.Exists(data))
+            {
+                foreach (FileInfo file in folderDirectory.GetFiles())
+                {
+                    file.Delete();
+                }
+
+                Directory.Delete(data);
+            }
```

Comment:
Should a test delete all the data we already have (for example we already scanned 3 receipts, we start the unit test, and we lose all the data from the scanned receipts)